htdocs/compta/facture/card.php

```diff
@@ -2948,15 +2948,12 @@
 			print '</div></div>';
 
 			// Next situation invoice
-			$opt = $form->selectSituationInvoices(GETPOST('originid'), $socid);
 			print '<div class="tagtr listofinvoicetype"><div class="tagtd listofinvoicetype">';
 			$tmp='<input type="radio" name="type" value="5"' . (GETPOST('type') == 5 && GETPOST('originid') ? ' checked' : '');
-			if ($opt == ('<option value ="0" selected>' . $langs->trans('NoSituations') . '</option>') || (GETPOST('origin') && GETPOST('origin') != 'facture' && GETPOST('origin') != 'commande')) $tmp.=' disabled';
+			if (GETPOST('origin') != 'facture' && GETPOST('origin') != 'commande') $tmp.=' disabled';
 			$tmp.= '> ';
 			$text = $tmp.$langs->trans("InvoiceSituationAsk") . ' ';
-			$text .= '<select class="flat" id="situations" name="situations">';
-			$text .= $opt;
-			$text .= '</select>';
+			$text .= $form->selectSituationInvoices(GETPOST('originid', 'int'), $socid);
```

Comment:
Where are the <select> around the <option> now ?

Reply:
@eldy 
if you speak of the "select" tags they have been moved in the `selectSituationInvoices()` function